src/test/java/com/puppycrawl/tools/checkstyle/api/TokenTypesTest.java

```diff
@@ -29,9 +29,8 @@
 public class TokenTypesTest {
     @Test
     public void testGetShortDescription() {
-        assertEquals("short description for EQUAL",
-                "The <code>==</code> (equal) operator.", TokenUtils
-                        .getShortDescription("EQUAL"));
+        assertEquals("short description for EQUAL", "The ",
+                TokenUtils.getShortDescription("EQUAL"));
```

Comment:
This is not clear ...
as we changed first sentence to be "The {@code ==} (equal) operator." in javadoc.
so `TokenUtils.getShortDescription("EQUAL")` should return `The {@code ==} (equal) operator.`

before fix it was:
```
$ mvn clean compile
....
$ grep "^EQUAL" target/classes/com/puppycrawl/tools/checkstyle/api/tokentypes.properties
EQUAL=The <code>==</code> (equal) operator.
```
after fix:
```
$ mvn clean compile
....
$ grep "^EQUAL" target/classes/com/puppycrawl/tools/checkstyle/api/tokentypes.properties
EQUAL=The 
```

That is not acceptable.